src/plugins/MamicoCoupling.h

```diff
@@ -0,0 +1,62 @@
+#ifndef SRC_PLUGINS_MAMICOCOUPLING_H_
+#define SRC_PLUGINS_MAMICOCOUPLING_H_
+
+#include "PluginBase.h"
+#ifdef MAMICO_COUPLING
+#include <climits> //for INT_MAX in indexconversion
+#include <coupling/services/MacroscopicCellService.h>
+#include <coupling/interface/MamicoInterfaceProvider.h>
+#include <coupling/interface/impl/ls1/LS1RegionWrapper.h>
+//#include <MacroscopicCellService.h>
+//#include <MamicoInterfaceProvider.h>
+#endif
+
+//#include "particleContainer/LinkedCells.h"
+
+
+class MamicoCoupling: public PluginBase {
+
+public:
+	MamicoCoupling() {}
+	virtual ~MamicoCoupling() {}
+
+	void init(ParticleContainer* particleContainer,
+			DomainDecompBase* domainDecomp, Domain* domain);
+
+    void readXML(XMLfileUnits& xmlconfig);
```

Comment:
```suggestion
    void readXML(XMLfileUnits& xmlconfig) override;
```
Missing at all overridden functions